src/redux/actions/genes/loadMarkerGenes.js

```diff
@@ -0,0 +1,47 @@
+import {
+  MARKER_GENES_ERROR, MARKER_GENES_LOADING,
```

Comment:
Parform changes commented in https://github.com/biomage-ltd/ui/pull/396/files#r673322458